client/lib/analytics/tracker-buckets.ts

```diff
@@ -52,6 +53,7 @@ export const AdTrackersBuckets: { [ key in AdTracker ]: Bucket | null } = {
 	bing: Bucket.ADVERTISING,
 	floodlight: Bucket.ADVERTISING,
 	googleAds: Bucket.ADVERTISING,
+	googleTagManager: Bucket.ADVERTISING,
```

Comment:
Just to make sure, that will require standalone consent given on checkout pages to load the script. Is it the way we want to proceed?

Will we still try cross-domain consent at a later stage?

Reply:
I've asked the question in the discussion thread, and indeed we do not proceed with cross-domain consent at this time. It might involve more legal and technical adjustments in the near future.

Reference:
pau2Xa-4J9-p2#comment-13342